src/api/methods/getUnsignedTransferTx.js

```diff
@@ -0,0 +1,15 @@
+const makeTransfer = require('../../txHelpers/makeTransfer');
+
+module.exports = async (bridgeState, from, to, color, value) => {
+  const tx = await makeTransfer(
+    {
+      balances: bridgeState.currentState.balances,
+      unspent: bridgeState.currentState.unspent,
+    },
+    from,
+    to,
+    value,
+    color
+  );
+  return tx.toJSON();
```

Comment:
@troggy, should this new test added yet? Because it is provided to guard leap-core bug, may be the right time is after the fix? Otherwise unittest of  getUnsignedTransferTx will always fail.

Reply:
on a second thought, the test is not necessary here, once everything is done properly and tested on leap-core side. So this PR is fine 👍, just need to add fixed leap-core once it fixed